emission/core/wrapper/entry.py

```diff
@@ -182,6 +182,9 @@ def create_fake_entry(user_id, key, data, write_ts, create_id=False):
       result_entry.user_id = user_id
       result_entry.metadata = ecwm.Metadata.create_metadata_for_fake_result(key, write_ts)
       result_entry.data = data
+      #necessary values required by forest model
+      result_entry['data']['start_local_dt']=result_entry['metadata']['write_local_dt']
+      result_entry['data']['end_local_dt']=result_entry['metadata']['write_local_dt']
```

Comment:
wait what?! the trip's start and end timestamps are almost always guaranteed to not be the same as the time it was created. And by putting this into `core/wrapper/entry.py`, you are adding this to every single entry of any type, created in any way.

This is just wrong.

Reply:
This is done in `create_fake_entry` function, which is used only for testing purpose

```
$ grep -rl create_fake_entry | grep -v __pycache__
./emission/core/wrapper/entry.py
./emission/tests/modellingTests/modellingTestAssets.py
```

Since `start_local_dt` and `end_local_dt' components were not required for greedy binning, they were not being added to the trips added via `generate_mock_trips`. For Random Forest model pipeline, they need to be part of the `Entry` data that we generate.

Since `TestRunRandomForest` (which is just the first test file written) simply checks the running of the pipeline, any value for now would suffice.

These values are bound to evolve as we keep adding more and more tests as per the requirements.